src/allmydata/client.py

```diff
@@ -175,8 +175,6 @@ def generate(self):
         """I return a Deferred that fires with a (verifyingkey, signingkey)
         pair. The returned key will be 2048 bit"""
         keysize = 2048
-        # RSA key generation for a 2048 bit key takes between 0.8 and 3.2
-        # secs
```

Comment:
Instead of updating this one I just took it out.  I don't see the point of commenting on the performance cost of this one operation specifically.